test/Datadog.Trace.ClrProfiler.Managed.Tests/Datadog.Trace.ClrProfiler.Managed.Tests.csproj

```diff
@@ -10,4 +10,10 @@
     <ProjectReference Include="..\Datadog.Trace.TestHelpers\Datadog.Trace.TestHelpers.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Web">
+      <HintPath>..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.6.1\System.Web.dll</HintPath>
```

Comment:
Having the `<HintPath>` here is weird, especially since it includes a specific framework version. If you had to add this so the `netcoreapp2.1` wouldn't fail, it's because `System.Web` should only be included when building for .NET Framework and not for .NET Core or .NET Standard:

```xml
<ItemGroup Condition=" '$(TargetFramework)' != 'netcoreapp2.1' ">
    <Reference Include="System.Web" />
</ItemGroup>
```